samples/matter/common/src/dfu/ota/ota_util.cpp

```diff
@@ -21,18 +21,19 @@
 using namespace chip;
 using namespace chip::DeviceLayer;
 
+namespace Nrf::Matter
```

Comment:
Why do we want to keep these static members in Nrf::Matter and not anonymous namespace?

Reply:
Actually, I repaired a build when the OTA was disabled and BLE DFU was enabled, because the brackets were placed wrongly, but I can move the bracket at the bottom instead of moving the namespace declaration.